modules/oc-mirror-troubleshooting-v2.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-troubleshooting-v2_{context}"]
+== Troubleshooting oc-mirror plugin v2 errors
+
+oc-mirror plugin v2 now logs all image mirroring errors in a separate file, making it easier to track and diagnose failures.
```

Comment:
Hi @subhtk 

we might want to say that oc-mirror v2 behavior is different when errors are encountered based on the type of image: 

errors mirroring release images, or release component images are considered fatal (not sure this is the right word): A failure in mirroring a release image or release component image will lead to stopping the mirroring process immediately. 
On the other hand, errors during mirroring of operators, operator related images or additional images will not stop the mirroring, being day 2 operators.... Therefore, mirroring continues, and the user is notified by the console log (message every 8 images to recap the number of errors encountered so far), and by the final console message which advises the user to take a look at the log file (mirroring_errors_date_time.log)

Reply:
Added an admonition as discussed.